test/net-test.js

```diff
@@ -283,6 +283,113 @@ describe('Net', function() {
+        assert.equal(pkt.type, packets.types.GETCFHEADERS);
+
+        assert.equal(pkt.startHeight, 12);
+        assert.equal(pkt.stopHash.toString('hex'), 'ab'.repeat(32));
+        assert.equal(pkt.filterType, common.FILTERS.BASIC);
+        assert.equal(pkt.getSize(), 37);
+        assert.bufferEqual(pkt.toRaw(), Buffer.from('000c000000abababababababababababababababababababababababababababababababab', 'hex'));
+      };
+
+      let pkt = new packets.GetCFHeadersPacket(0, 12, Buffer.from('ab'.repeat(32), 'hex'));
+      check(pkt);
+
+      pkt = packets.GetCFHeadersPacket.fromRaw(pkt.toRaw());
+      check(pkt);
+    });
+
+    it('getCFCheckpt', () => {
+      const check = (pkt) => {
+        assert.equal(pkt.cmd, 'getcfcheckpt');
+        assert.equal(pkt.type, packets.types.GETCFCHECKPT);
+
+        assert.equal(pkt.filterType, common.FILTERS.BASIC);
+        assert.equal(pkt.getSize(), 33);
+        assert.bufferEqual(pkt.toRaw(), Buffer.from('00abababababababababababababababababababababababababababababababab', 'hex'));
+      };
+
+      let pkt = new packets.GetCFCheckptPacket(0, Buffer.from('ab'.repeat(32), 'hex'));
+      check(pkt);
+
+      pkt = packets.GetCFCheckptPacket.fromRaw(pkt.toRaw());
+      check(pkt);
+    });
+
+    it('CFilter', () => {
+      const check = (pkt) => {
+        assert.equal(pkt.cmd, 'cfilter');
+        assert.equal(pkt.type, packets.types.CFILTER);
+
+        assert.equal(pkt.filterType, common.FILTERS.BASIC);
+        assert.equal(pkt.getSize(), 94);
+        assert.bufferEqual(pkt.toRaw(), Buffer.from('00abababababababababababababababababababababababababababababababab3cabababababababababababababababababababababababababababababababababababababababababababababababababababababababababababab', 'hex'));
+      };
+
+      let pkt = new packets.CFilterPacket(0, Buffer.from('ab'.repeat(32), 'hex'), Buffer.from('ab'.repeat(60), 'hex'));
+      check(pkt);
+
+      pkt = packets.CFilterPacket.fromRaw(pkt.toRaw());
+      check(pkt);
+    });
+
+    it('CFHeaders', () => {
+      const check = (pkt) => {
+        assert.equal(pkt.cmd, 'cfheaders');
+        assert.equal(pkt.type, packets.types.CFHEADERS);
+
+        assert.equal(pkt.filterType, common.FILTERS.BASIC);
+        assert.equal(pkt.getSize(), 226);
+        assert.bufferEqual(pkt.toRaw(), Buffer.from('00ababababababababababababababababababababababababababababababababaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa05abababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababababab', 'hex'));
+      };
+
+      let pkt = new packets.CFHeadersPacket(0, Buffer.from('ab'.repeat(32), 'hex'), Buffer.from('aa'.repeat(32), 'hex'), Array(5).fill(Buffer.from('ab'.repeat(32), 'hex')));
```

Comment:
same kind of style note here

```suggestion
      let pkt = new packets.CFHeadersPacket(
        0,
        Buffer.from('ab'.repeat(32), 'hex'),
        Buffer.from('aa'.repeat(32), 'hex'),
        Array(5).fill(Buffer.from('ab'.repeat(32), 'hex'))
      );
```